readthedocs/payments/utils.py

```diff
@@ -7,15 +7,20 @@
 
 import stripe
 import structlog
-from django.conf import settings
+from djstripe.models import APIKey
 
 
-stripe.api_key = settings.STRIPE_SECRET
 log = structlog.get_logger(__name__)
 
 
+def get_stripe_api_key():
+    """Return Stripe API key defined in the dj-stripe database."""
+    return APIKey.objects.filter(type="secret").first().secret
+
+
 def delete_customer(customer_id):
     """Delete customer from Stripe, cancelling subscriptions."""
+    stripe.api_key = get_stripe_api_key()
```

Comment:
Not sure if I like this pattern of defining the API key every time we need to interact with stripe outside djstripe, looks like we should just migrate to the new pattern of having a client instance, instead of defining a get_stripe_api_key function, we can have a `get_stripe_client()` function.